PencilCase/PencilCase/scripts/CustomBindings/ListviewBinding.ts

```diff
@@ -0,0 +1,12 @@
+﻿class ListviewBinding implements KnockoutBindingHandler {
```

Comment:
Implemented listview custom binding handler. 
Due to dynamically inserted(KO binding) DOMs are not enhanced by JQM, so need this handler here, once the product collection changed, re-enhance the DOM.

Note: Note that I did add any export keyword in this class and the Binding class in Binding.ts. In this way, it makes the members in this file are globally available. Just like the JQuery, KO libraries.